TPG26x/iocBoot/iocTPG26x-IOC-01/config.xml

```diff
@@ -3,8 +3,8 @@
     <config_part>
         <ioc_desc>Pfeiffer TPG 26X pressure sensor</ioc_desc>
         <macros>
-            <macro name="PORT" pattern="^COM[0-9]+$" description="Serial COM Port" hasDefault="NO" />
-            <macro name="BAUD" pattern="^9600|19200|38400$" description="Baud rate, must be either 9600, 19200 or 38400 (default: 9600)" defaultValue="9600" hasDefault="YES" />
+            <xi:include href="../../../COMMON/PORT.xml" />
+            <xi:include href="../../../COMMON/BAUD9600.xml" />
```

Comment:
The baud rates offered in BAUD9600 are not compatible with the device, probably best to leave as is